CHANGELOG.md

```diff
@@ -3,6 +3,109 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.0-beta.0
+
+_Jan 25, 2024_
+
+We are glad to announce MUI X v7 beta!
+This version comes with a number of improvements, bug fixes and exciting features 🎉:
+
+### 🚀 Release the [Date Time Range Picker](https://next.mui.com/x/react-date-pickers/date-time-range-picker/) component (#9528) @LukasTy
```

Comment:
Wasn't sure how to incorporate the bigger descriptions in the release highlights, so I used H3s, but I'd love notes on this 👍

Reply:
I  would always stick with the same structure for highlights: a list of elements instead of h3 titles
But I'd be curious to see what the other thinks

Reply:
Yeah, I agree, let's keep it consistent.
I'm not sure that changing the structure just for some special one-off highlights is worth the "disruption of flow". 🤔